src/raster/rasterband.rs

```diff
@@ -260,6 +261,18 @@ impl<'a> RasterBand<'a> {
         Ok(())
     }
 
+    pub fn get_color_interpretation(&self) -> GDALColorInterp::Type {
```

Comment:
in rust the "get_" prefix is usually skipped. So please remove it from the getter methods.